include/swift/AST/Module.h

```diff
@@ -209,7 +210,20 @@ class ModuleDecl : public DeclContext, public TypeDecl {
 
 public:
   typedef ArrayRef<Located<Identifier>> AccessPathTy;
-  typedef std::pair<ModuleDecl::AccessPathTy, ModuleDecl*> ImportedModule;
+  
+  typedef struct _ImportedModule {
+    /// neither of these can be const because some things update after construction
+    ModuleDecl::AccessPathTy accessPathTy;
+    ModuleDecl * moduleDecl;
```

Comment:
Nitpick: We follow the convention of having the `*` next to the RHS, so this should be `ModuleDecl *moduleDecl;`.

Reply:
Resolved in af9a745, I hope. Let me know if you see any more of them.